uranium/scripts/uranium_standalone

```diff
@@ -184,6 +187,26 @@ def _get_site_file_path(venv_directory):
     ).communicate()[0].decode('utf-8').rstrip('c\n')
 
 
+def _override_pydistutils_prefix(install_dir):
+    """
+    write a .pydistutils.cfg file to override the prefix variable
+    """
+    if os.name == 'posix':
+        env_distutils_filename = '.pydistutils.cfg'
+    else:
+        env_distutils_filename = 'pydistutils.cfg'
+
+    if not os.path.exists(install_dir):
+        os.makedirs(install_dir)
+    env_distutils_file = os.path.join(install_dir, env_distutils_filename)
+
+    with open(env_distutils_file, 'w') as fh:
```

Comment:
Would it be worth handling the case where the file already exists? For the sprinter script it is not really possible.